roles/kubernetes/master/tasks/kubeadm-secondary-experimental.yml

```diff
@@ -25,6 +25,24 @@
     port: "{{kubeadm_discovery_address.split(':')[1]}}"
     timeout: 180
 
+
+- name: Upload certificates so they are fresh and not expired
+  command: >-
+    {{ bin_dir }}/kubeadm init phase
+    --config {{ kube_config_dir}}/kubeadm-config.yaml
+    upload-certs --experimental-upload-certs
+    {% if kubeadm_certificate_key is defined %}
+    --certificate-key={{ kubeadm_certificate_key }}
```

Comment:
It's still hardcoded in:

tests/files/packet_centos7-flannel-addons.yml
tests/files/gce_centos7-flannel-addons.yml

And it's still getting auto created on deployment (that's intentional right?) in inventory/sample/group_vars/k8s-cluster/k8s-cluster.yml

Reply:
yes it's intentional